internal/tasks/manifests.go

```diff
@@ -733,6 +740,11 @@ func (j *Janitor) doVulnerabilityCheck(account keppel.Account, repo keppel.Repos
 				return fmt.Errorf("Clair reports indexing of %s as finished, but vulnerability report is 404", manifest.Digest)
 			}
 			vulnStatuses = append(vulnStatuses, clairReport.VulnerabilityStatus())
+
+			if vulnInfo.IndexFinishedAt == nil {
+				now := j.timeNow()
```

Comment:
We already knew `IsIndexed` before starting `GetVulnerabilityReport`, so we can use the same `now` as before.